functions/fundle.fish

```diff
@@ -123,7 +123,12 @@ function __fundle_check_date -d "check date"
 end
 
 function __fundle_get_url -d "returns the url for the given plugin" -a repo
-	echo "https://github.com/$repo.git"
+    set tag  (echo "$repo" | cut -s -d'@' -f2)
+    set repo (echo "$repo" | cut -d'@' -f1)
+    set url "https://github.com/$repo.git"
```

Comment:
Faster and fishier would be to use the delightful [`string`](https://fishshell.com/docs/current/cmds/string.html) builtin.

Reply:
You're right ! I'll change that.